frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.html

```diff
@@ -268,7 +268,8 @@ <h3>Cutting is prohibited within:</h3>
     <app-error-message fieldId="accept-rules-error" name="To purchase a tree permit, agreeing to the rules" [control]="applicationRulesForm.controls.acceptRules"></app-error-message>
   </div>
   </div>
-    <button id="submit-application" class="usa-button-primary usa-button-big" (click)="createApplication()" type="submit">Continue to Checkout</button>
+    <button *ngIf="!buttonDisabled" id="submit-application" class="usa-button-primary usa-button-big" (click)="createApplication()" type="submit">Continue to Checkout</button>
+    <button *ngIf="buttonDisabled" id="submit-application" class="usa-button-primary usa-button-disabled usa-button-big" type="button">Continue to Checkout</button>
```

Comment:
This works but we can clean it up by leveraging our framework here. With angular's [binding syntax](https://angular.io/guide/binding-syntax#data-binding-and-html) we can simply add `[disabled]="buttonDisabled"` to the `<button>`.

This consolidates two lines back into one as well as avoids the use of `ngIf` which will directly manipulate the DOM by adding and removing elements, which as a general rule of thumb is more resource intensive for the browser.

Reply:
Sweet! I like that. Fixing it now.